Cargo.toml

```diff
@@ -1119,6 +1119,11 @@ wasm = false
 name = "scene"
 path = "examples/scene/scene.rs"
 
+# Scene Building
```

Comment:
I added an example in order to test that it works out, but I am not entirely sure it's necessary. Feel free to tell me to remove it.